apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -451,7 +451,7 @@ export class ChangeAction extends BaseRushAction {
           console.log(`Not overwriting ${filePath}...`);
           return Promise.resolve();
         }
-      });
+      }).catch(console.error);
```

Comment:
>.catch(console.error); [](start = 8, length = 22)

The TSLint rule is flagging an actual oversight here.

Here we need real error handling.  Maybe something like:

```
catch ((error) => {
  console.error(colors.red(error.message));
});
```